sway/handlers.c

```diff
@@ -104,6 +108,8 @@ static void handle_output_destroyed(wlc_handle output) {
 	}
 	if (i < list->length) {
 		destroy_output(list->items[i]);
+	} else {
+		return;
```

Comment:
Sorry, didn't expand the diff. Thought this was part of `handle_output_created`.